src/components/ItemLookup.vue

```diff
@@ -56,8 +56,20 @@ const onOptionSelected = ( value: unknown ) => {
 };
 
 const debouncedSearchForItems = debounce( async ( debouncedInputValue: string ) => {
+	const regExp = new RegExp( `\\b${escapeRegExp( debouncedInputValue )}`, 'i' );
```

Comment:
I wonder if we really should debounce the filtering of the static list. But it probably doesn't hurt :thinking:

Reply:
I was wondering that as well… I could imagine people asking for both (possibly depending on their reduced-motion preference?). I’d leave it like this for now (simpler code) and see if product or UX have anything to say. (Subjectively, it still feels fast enough to me.)